expenses-manager-backend/src/main/java/com/endava/expensesmanager/service/impl/ExpenseServiceImpl.java

```diff
@@ -88,5 +89,12 @@ private boolean validateDates(LocalDateTime startDate, LocalDateTime endDate) {
         return false;
     }
 
+    @Override
+    public BigDecimal getTotalAmountByDateBetween(int userId, LocalDateTime startDate, LocalDateTime endDate) {
+        List<ExpenseDto> expenses = getExpensesByUserId(userId, startDate, endDate);
```

Comment:
startDate and endDate are not required in controller, what will happen if are null here?